drupal/manifests/drupal-secrets.yaml

```diff
@@ -0,0 +1,11 @@
+#helm:generate helmc tpl -d tpl/values.toml -o manifests/drupal-secrets.yaml $HELM_GENERATE_FILE
+apiVersion: v1
+kind: Secret
+metadata:
+  name: drupal
+  labels:
```

Comment:
I'd just be consistent with the other kind of resources which follow (please correct me if I am wrong) [our guidelines](https://github.com/bitnami/charts/blob/master/_docs/authoring-helm-charts.md#labels)

We need to revisit this whenever we migrate these charts for the new helm, there is a lot of work being done now related to namespaces. https://github.com/kubernetes/helm/issues/838